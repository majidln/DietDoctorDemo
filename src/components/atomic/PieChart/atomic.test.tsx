import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {PieChart} from './index';

it('PieChart render correctly', () => {
  renderer.create(<PieChart data={[55, 45]} />);
});

it('PieChart Snapshot renders correctly', () => {
  const tree = renderer.create(<PieChart data={[55, 45]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
