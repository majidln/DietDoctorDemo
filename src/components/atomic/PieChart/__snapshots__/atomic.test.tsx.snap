// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PieChart Snapshot renders correctly 1`] = `
<ChartView
  data={
    Array [
      55,
      45,
    ]
  }
/>
`;
